### PR TITLE
Feature/use tabs in app developer secret

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

**Review by PierreJeanjacquot — APPROVED** [review]
:+1:

**Review by james-toussaint — APPROVED** [review]
Thank you :+1:

File: for-developers/confidential-computing/app-developer-secret.md [hunk 1]
```diff
@@ -160,28 +161,31 @@ In this section, you will:
 
 Create the `Dockerfile`
 
-- for a Javascript application
-  ```bash
-  # Starting from a base image supported by SCONE
-  FROM node:14-alpine3.11
+### For a Javascript application
```
Comment (mcornaton) [inline for-developers/confidential-computing/app-developer-secret.md:164]: No "tab in tab" possibility?

File: for-developers/confidential-computing/app-developer-secret.md [hunk 2]
```diff
@@ -201,78 +205,92 @@ IMG_TO=<docker-hub-user>/${IMG_NAME}:1.0.0-debug
...
+
+```bash
+iexec app check-secret --chain bellecour
+```
+
+### Run the TEE app
+
+{% tabs %}
+{% tab title="Scone" %}
+[Run your application](create-your-first-sgx-app.md#run-the-tee-app).
```
Comment (mcornaton) [inline for-developers/confidential-computing/app-developer-secret.md]:
```suggestion
[Run your application](create-your-first-sgx-app.md#run-the-tee-app)
```
Other single lines in tabs don't end with a point.

**Review by mcornaton — APPROVED** [review]
Thanks for this increment!

File: for-developers/confidential-computing/app-developer-secret.md (hunk 1 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/app-developer-secret.md:164]: sadly, it is not possible

Comment (jbern0rd) [reply for-developers/confidential-computing/app-developer-secret.md:164]: Nested features are limited with gitbook

File: for-developers/confidential-computing/app-developer-secret.md (hunk 2 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/app-developer-secret.md]: Thanks, removed in bbc38ab

**Pull request merged**